Yoast/wordpress-seo
# Fro 123 remove wpseo admin help panel class

## Context


* Remove the unused WPSEO_Admin_Help_Panel class

## Summary


This PR can be summarized in the following changelog entry:

* Remove the unused WPSEO_Admin_Help_Panel class and javascript supporting this class.

## Relevant technical choices:

* Before the html was restyled to the new styling it used the `.yoast-help-button` to activate the javascript. But this won't be used anymore so the javascript can be removed.

## Test instructions

This PR can be tested by following these steps:

* Pull this branch
* Pull the develop branch from the javascript repo
* Link the javascript branch
* Build the plugin
* Check in the post editor and on the settings pages if there are no errors

## UI changes

## Documentation

## Quality assurance


Fixes https://yoast.atlassian.net/browse/FRO-123
